docs/overlay.html

```diff
@@ -376,8 +376,7 @@ <h3 class="tm-article-subtitle">Example</h3>
 
                                 <li class="uk-width-medium-1-3">
                                     <figure class="uk-overlay uk-overlay-hover">
-                                        <img src="images/placeholder_600x400.svg" width="600" height="400" alt="Image">
-                                        <figcaption class="uk-overlay-panel uk-overlay-background  uk-overlay-bottom uk-overlay-slide-bottom">
+                                        <img src="images/placeholder_600x400.svg" width="600" height="400" alt="Image"><figcaption class="uk-overlay-panel uk-overlay-background uk-overlay-bottom uk-overlay-slide-bottom">
```

Comment:
was receiving Trailing Whitespace somewhere around there, so I think I brought figcaption up for that. do I move figcaption tag to a separate line?